site.yml

```diff
@@ -14,6 +14,18 @@
   - { role: base }
   - { role: serf }
   - { role: dev }
+  - { role: vagrant }
+
+- hosts: build
```

Comment:
I think you're still going to need the `build` host group to isolate `devtest` from vagrant deps